lstchain/scripts/lstchain_merge_hdf5_files.py

```diff
@@ -72,6 +72,11 @@
     help='Skip checks when merging files'
 )
 
+parser.add_argument(
+    '--keys-to-copy',
+    nargs="+", default=[''],
```

Comment:
This should be `nargs="*"` (allow it to be empty) and the default is then an empty list.

The default should just be an empty list, not a list with one element that is an empty string.